code/game/machinery/autolathe/autolathe_datums.dm

```diff
@@ -59,6 +59,11 @@
 	path = /obj/item/device/multitool
 	category = "Tools"
 
+/singleton/autolathe_recipe/geiger
+	name = "geiger counter"
+	path = /obj/item/device/geiger
```

Comment:
PSA: At the moment this would make printing them free - Not necessarily an error per se, but probably unwanted by the maintainers